### DISCUSSION TITLE
FCP to Davinci Resolve - "Roundtrip Dummy" import issue

### DISCUSSION THREAD

**Discussion by DmytroNorth** [latenitefilms/BRAWToolbox#49] [General]
Hi Chris,
First of all thanks for creating such an amazing and much-anticipated tool!
You mentioned in the description that fcp project could be imported to Davinci Resolve with the original braw footage for further color grading. Could you please clarify how to achieve it?
When I am trying to import fcpxml I have the correct timeline. Audio is imported automatically and plays back as it should. However, no video (braw clips) are imported, and instead, they are reflected in braw as blacked-out "Roundtrip Dummy" clips.
<img width="1280" alt="Screenshot 2023-01-29 at 14 00 31" src="https://user-images.githubusercontent.com/20174979/215327812-fdaa628a-9306-4eda-9174-ebcb93a33f2d.png">

**Comment by latenitefilms** [comment]
See:

https://brawtoolbox.io/toolbox/#convert-project-to-resolve-friendly-fcpxml

**Reply by DmytroNorth** [reply]
Thanks a lot. It worked beautifully!
I did another test in a slightly different way and got another issue. When I "Break Apart Clip Items" in FCP and move it to Resolve it gives a "File not found" error for braw clips.
Is it just a limitation for now that braw files have to stay as clips synced with audio for the successful move to Resolve?
<img width="1280" alt="Screenshot 2023-01-29 at 14 22 57" src="https://user-images.githubusercontent.com/20174979/215329545-44694164-cb89-4c66-8fec-081899790cd4.png">
<img width="1280" alt="Screenshot 2023-01-29 at 14 22 46" src="https://user-images.githubusercontent.com/20174979/215329551-adb23d1d-af5a-4bae-9de9-241f758aae01.png">

**Reply by latenitefilms** [reply]
Yes, you can't break apart a BRAW Toolbox Synchronised Clip, otherwise it will break things in Final Cut Pro.
